libs/utils/utils.js

```diff
@@ -452,18 +453,17 @@ async function loadPostLCP(config) {
   loadFonts(config.locale, loadStyle);
 }
 
-export async function loadDeferred(area, blocks) {
-  if (getMetadata('nofollow-links') === 'on') {
-    const path = getMetadata('nofollow-path') || '/seo/nofollow.json';
-    const { default: nofollow } = await import('../features/nofollow.js');
-    nofollow(path, area);
-  }
+export async function loadDeferred(area, blocks, config) {
+  const path = `${config.contentRoot || ''}${getMetadata('links-path') || '/seo/links.json'}`;
+  const { default: setupMappedLinks } = await import('../features/links.js');
+  setupMappedLinks(path, area);
```

Comment:
You may be able to `import.then` this instead of awaiting.  Not sure it makes any/much difference but would allow for samplerum below to also start loading async.

Reply:
good idea - in general I think we should be using await less in our code